src/open_inwoner/openklant/services.py

```diff
@@ -1041,6 +1041,20 @@ def find_organisatie_for_kvk_and_vestiging(
 
         return self.client.partij.retrieve(vestiging_pi["identificeerdePartij"]["uuid"])
 
+    def get_partij_for_user(self, user: User) -> Partij | None:
+        if user.kvk:
```

Comment:
```
if user.kvk:
    return self.find_organisatie_for_kvk_and_vestiging(
    kvk=user.kvk,
    vestigingsnummer=user.vestiging,
    )
```

Shouldn't be necessary to check `if organisatie` since both this and the called method return `Partij | None`.